.github/workflows/ci.yml

```
@@ -15,7 +15,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        version: [1.68, stable]
+        version: [stable, nightly]
```

I completely agree with you that we should run tests not only on the "nightly" version but also on the "stable" version. If we want to support version 1.68 as the Minimum Supported Rust Version (MSRV), it would be best to include it in the CI.
```suggestion
        version: [1.68, stable, nightly]
```

---

Yes, I stated above, it causes [dependency version conflicts](https://github.com/estie-inc/cder/actions/runs/10199238160). (in particular, tokio in this case)

Maybe we could lock the dependency versions only for 1.68 testing, and use the latest crates for the stable/nightly, (some changes are required with CI matrixes settings..?)

---

Okay, we no longer support version 1.68. In that case, we might want to set another version as the MSRV. Which version should we specify as the new MSRV?

---

I think there are two options:
- Specify version 1.70.0 as the new MSRV, which aligns with [the latest version of tokio (1.39.2)](https://crates.io/crates/tokio/1.39.2).
- Use [tokio version 1.38.1](https://crates.io/crates/tokio/1.38.1), whose MSRV is 1.63.0. Since we use it only for testing, this might not be a problem.

---

@kenkoooo-san

Thanks for your advices!
I'm not very keen on updating MSRV for testing reason - we might have to update our CI everytime when tokio 1.xx changes its MSRV support. (which looks [quite frequent](https://github.com/tokio-rs/tokio?tab=readme-ov-file#supported-rust-versions)).

I thought it should be nice to specify tokio 1.38.1 for MSRV testing only, while using the latest minor versions of dependencies for the latest stable rust. Would that make sense to you..?

---

(it seems I'm not having maintainership of this crate any more, so it would be great if you can approved it to get it merged. Thank you in advance..!)